app/Main.hs

```diff
@@ -96,5 +100,6 @@ main = do
   home <- getHomeDirectory
   let ctx = Context pwd home
   progVersion <- progNameVersionTag
+  loadStdLibs
```

Comment:
Is this just used to donwload it upon any action, or do we ingest it into memory via this somehow?

Reply:
We check they are installed first
```
loadStdLibs :: IO ()
loadStdLibs = do
  success <- localStdLibs
  unless success downloadStdLibs
```

Reply:
Checking the diffs, this `loadStdLibs` gets run upon any command (it's been propagated to test main functions as well). I don't think it's a good decision to have the compiler grab files online if some directory does not exist. I'd be more comfortable wtih it if it were a package manager as there yes you have to interact with the internet.

However since we are building from source and not distributing a binary, the make install command can cover this case. 

Upon the files not being there we can repurpose the injections to tell the users to run a command, and even have a restart prompt with the following options

```
1: grab files from local source

2. grab online automatically

3. error and don't continue

4. continue at your own risk 
```
Pressing `1` prompts the user with giving a path, if no path is given it shows a default of the current path

Reply:
That can be a separate PR